### DISCUSSION TITLE
Include emissions costs in objective

### DISCUSSION THREAD

**Discussion by smryania** [NREL/REopt-API-Analysis#143] [6 - Emissions Questions] [answered]
In early March, 2024, there was an option in the web tool to check a box under Renewable Energy and Emissions to include climate costs in the objective.  I can not find this any longer. Did it move or was it removed?

**Comment by adfarth** [comment, marked as answer]
This input has moved to the "Clean Energy" accordion! This appears when you select "Clean Energy" as a goal in Step 2.
